internal/controllers/cryostat_controller_test.go

```diff
@@ -590,6 +599,12 @@ var _ = Describe("CryostatController", func() {
 						err := t.Client.Get(context.Background(), types.NamespacedName{Name: expectedLink.Name}, link)
 						Expect(kerrors.IsNotFound(err)).To(BeTrue())
 					})
+					It("should remove the application url from APIServer AdditionalCORSAllowedOrigins", func() {
+						apiServer := &configv1.APIServer{}
+						err := t.Client.Get(context.Background(), types.NamespacedName{Name: "cluster"}, apiServer)
+						Expect(kerrors.IsNotFound(err)).To(BeFalse())
```

Comment:
I think in this case we should expect err to not be nil.

Reply:
I think the error should be nil here, because we expect the APIServer to remain present after the cryostat CR is deleted, right?

Reply:
Oops, yes I meant it should be nil.